components/empty-state/demo/empty-state.html

```diff
@@ -43,6 +43,15 @@ <h5>Empty State Action Link</h5>
 				</template>
 			</d2l-demo-snippet>
 
+			<h5>Empty State Only Button</h5>
```

Comment:
```suggestion
			<h5>Empty State Only Link</h5>
```
Did you want to add one for the button too?